app/views/content/life-as-a-teacher/explore-subjects/chemistry/_article.html.erb

```diff
@@ -1,46 +1,35 @@
 <div class="row inset">
   <section class="col col-720 col-space-l-top">
-    <h2 class="heading--box-blue">Why teach chemistry</h2>
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
     <div class="inset">
-      <p>Chemistry underpins everything, so you'll be amazed at the questions your pupils will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no two days are the same in the classroom.</p>
-      <p>In addition to the personal rewards of teaching chemistry, there are practical benefits too, including:</p>
-      <ul>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">generous and secure pension</a></li>
-        <li>more holiday days than people in many other professions</li>
-        <li>opportunities for progression</li>
-      </ul>
+      <p><%= @front_matter["subject"].titleize %> underpins everything, so you'll be amazed at the questions your pupils will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no two days are the same in the classroom.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/partials/teaching/benefits' %>
     </div>
   </section>
 
   <section class="col col-720">
-    <%= render Content::QuoteComponent.new(
-    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
-    name: "Chris Farr, teacher training adviser",
-    large: true
-  ) %>
+    <!-- Quote: Chris Farr -->
+    <%= render 'content/partials/quotes/quote_chris' %>
   </section>
 
   <section class="col col-720 col-space-l-top">
     <h2 class="heading--box-blue">Shape your pupils' futures</h2>
     <div class="inset">
       <p>Teaching chemistry opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from medicine and forensic science, to palaeontology and oceanography.</p>
```

Comment:
maybe replace "chemistry" with `<%= @front_matter["subject"] %>` for consistency